hanko-js/src/lib/HankoClient.ts

```diff
@@ -327,6 +330,18 @@ class WebauthnClient extends AbstractClient {
           reject(e);
         })
         .then((challenge: CredentialRequestOptionsJSON) => {
+          if (
+            challenge.publicKey.allowCredentials != undefined &&
+            this.isAndroidUserAgent
+          ) {
+            for (
+              let i = 0;
+              i < challenge.publicKey.allowCredentials.length;
+              i++
+            ) {
+              delete challenge.publicKey.allowCredentials[i].transports;
+            }
+          }
```

Comment:
Is it possible to simplify the loop? E.g.:
```suggestion
          if (this.isAndroidUserAgent) {
            challenge.publicKey?.allowCredentials?.map((c) => {
              delete c.transports;
            });
          }
```
```